graphql/queries/currentRules.js

```diff
@@ -0,0 +1,7 @@
+import gql from 'graphql-tag'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
import gql from "graphql-tag"
```